lamindb/_curate.py

```diff
@@ -311,6 +385,8 @@ def _update_registry_all(self, validated_only: bool = True, **kwargs):
     def validate(self, organism: str | None = None) -> bool:
         """Validate variables and categorical observations.
 
+        This method also registers the validated records in the current instance.
```

Comment:
Might need 1 more sentence of context when that happens? Validated against external instances or public ontologies? Else, they'd already be registered in the current instance, right?